src/averageAges.js

```diff
@@ -52,7 +71,31 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const ageGap = [];
+
+  const mothers = people.filter(person =>
+    people.some(child => child.mother === person.name));
```

Comment:
```suggestion
  const mothers = people.filter(
    person => people.some(child => child.mother === person.name)
  );
```
Не разрываем коллбек. Если переносим на новую строку - то переносим всю начинку, а не пополам